index.html

```diff
@@ -418,71 +417,21 @@
-          platform. This section provides examples of how a webapp can be designed to work with a
-          variety of hypothetical push services as needed.
-        </p>
-        <p>
-          The Push API provides several interface parameters and attributes that can enable a
-          webapp to deliver and obtain the necessary push service specific data enabling use of
-          that particular push service. These parameters and attributes include:
-        </p>
-        <ul>
-          <li>The <code>subscriptionId</code> of a <code>PushSubscription</code>, as an opaque
-          string to the Push API, may be used by push services to provide meaningful data to the
-          <a>application server</a> through the webapp. For example, the
-          <code>subscriptionId</code> may be used as a push service API request parameter or
-          request body element.
-          </li>
-          <li>As a URI, the <code>endpoint</code> of a <code>PushSubscription</code> provides a
-          flexible means for push services to deliver webapp-specific subscription and/or push
-          service API parameters to the <a>application server</a>, through the webapp.
-          </li>
-        </ul>
-        <p>
-          Push services that are compatible with the Push API will be expected to clarify how these
-          data are used in their APIs, e.g. through their developer programs. Examples of
-          considerations include:
+          The fields included in the <code>PushSubscription</code> is all the information needed for
+          an <a>application server</a> to send a <a>push message</a>. Push services that are
+          compatible with the Push API will be expected to provide an <a>endpoint</a> that conforms
+          to <em>TODO: refer to IETF spec</em>.  These parameters and attributes include:
         </p>
         <ul>
-          <li>The URI used by the <a>application server</a> to issue push service API requests,
-          which may be based upon the <code>endpoint</code>, and for a particular API request
-          require additional parameters e.g. the <code>subscriptionId</code>.
-          </li>
-          <li>The push service API request body, which may require that the <code>endpoint</code>
-          or <code>subscriptionId</code> be present in some body element.
-          </li>
-          <li>The <code>PushSubscription</code> attributes may only be used by the <a>application
-          server</a> to associate the webapp to a specific push service subscription, with the
-          details of the push service API relying upon other pre-configured data.
+          <li>The <code><a>endpoint</a></code> of a <code>PushSubscription</code> is a URI that
+          allows an <a>application server</a> to request delivery of a <a>push message</a> to
+          a <a>webapp</a> subscription.
           </li>
+          <li><em>TODO: More fields to add.</em></li>
```

Comment:
This item is not very useful. Please file specific issues for feature requests or bugs. No need to refer to those issues here either, unless they help resolve inconsistencies internal to this document.